front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/MatrixQuestion/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import { useTheme } from 'styled-components';
+
+import { ResultUngrouped } from 'api/survey_results/types';
+
+import useLocale from 'hooks/useLocale';
+
+import {
+  getTenantColourByPercentage,
+  getLinearScaleLabelsArray,
+  getPercentage,
+  getPercentageTextBorder,
+  getStatementsWithResultsArray,
+  StickyTd,
+  StledTh,
+} from './utils';
+
+type Props = {
+  result: ResultUngrouped;
+};
+
+const MatrixQuestion = ({ result }: Props) => {
+  const theme = useTheme();
+  const locale = useLocale();
+
+  // Create arrays for the statements & answers and linear scale
+  // labels so they are easier to work with.
+  const statementsWithResults = getStatementsWithResultsArray(result);
+  const linearScaleLabels = getLinearScaleLabelsArray(result);
+
+  return (
+    <Box pt="16px" overflowX="auto">
+      <table style={{ borderCollapse: 'separate', borderSpacing: '8px 8px' }}>
+        <thead>
+          <tr>
+            <th />
+            {linearScaleLabels.map((linearScaleMultiloc) => (
+              <StledTh key={linearScaleMultiloc.value}>
+                <Text m="0px">{linearScaleMultiloc.label[locale]}</Text>
+              </StledTh>
+            ))}
+          </tr>
+        </thead>
+
+        <tbody>
+          {statementsWithResults.map((statementWithResult) => (
+            <tr key={statementWithResult.statementKey}>
+              <StickyTd style={{ background: 'white' }}>
+                <Box display="flex" minHeight="80px">
+                  <Text m="0px" my="auto">
+                    {statementWithResult.statementMultiloc[locale]}
+                  </Text>
+                </Box>
+              </StickyTd>
+              <>
+                {linearScaleLabels.map((linearScaleLabel) => {
+                  // Get percentage of respondents who chose this value
+                  const percentage = getPercentage(
+                    statementWithResult,
+                    linearScaleLabel
+                  );
+
+                  return (
+                    <td
```

Comment:
Also here maybe we can use our existing components? In any case - better to not use inline styles.